client/apps/shipping-label/state/actions.js

```diff
@@ -492,6 +492,91 @@ export const updatePaperSize = ( value ) => {
 	};
 };
 
+const purchaseLabelResponse = ( response, error ) => {
+	return { type: PURCHASE_LABEL_RESPONSE, response, error };
+};
+
+const pollForLabelsPurchase = ( dispatch, getState, orderId, labels, error, showPrintDialog ) => {
```

Comment:
What do you think about changing this to expect an object instead of multiple arguments? I think it could make the calls easier to understand, especially around `labels`, `error`, and `showPrintDialog`. (like in `purchaseLabel()` and recursive calls inside the method.